src/pacmod3_core.cpp

```diff
@@ -26,6 +26,7 @@ const uint32_t AS::Drivers::PACMod3::ParkingBrakeCmdMsg::CAN_ID = 0x124;
 const uint32_t AS::Drivers::PACMod3::ShiftCmdMsg::CAN_ID = 0x128;
 const uint32_t AS::Drivers::PACMod3::SteerCmdMsg::CAN_ID = 0x12C;
 const uint32_t AS::Drivers::PACMod3::TurnSignalCmdMsg::CAN_ID = 0x130;
+const uint32_t AS::Drivers::PACMod3::DoorSignalCmdMsg::CAN_ID = 0x140;
```

Comment:
These are in alphabetical order for easier maintenance. Please move this up to right under the DashControlsRightCmdMsg.

Reply:
sure

Reply:
Sorry. I was wrong about this. They are in CAN ID order, not alpha. Ignore please.

Reply:
I was wrong, but I realized that it is still out of order. It should go after `WiperCmdMsg` because it is a higher CAN ID. Please fix.